### ISSUE TITLE
SQL update 616 (616_corepatch_12945.sql) drops spell_template table

### ISSUE THREAD

**Issue by grentech** [cmangos/wotlk-db#59]
Hello,

I just installed cmangos (Debian Linux) using a combination of the cmangos wiki and the referenced forum post (http://cmangos.net/thread-7481.html).

When the InstallFullDB.sh executes it also loads the updates one of which is 616_corepatch_12945.sql and it drops the spell_template table.  Which causes mangosd to fail to start.

In both the Wiki the forum post it is recommended to run InstallFullDB.sh as a last step (of the DB process).  

If that is the case then the 616 either needs to be removed or updated to not drop the table or clarification needs to be added as to what steps need to be taken after it is dropped.

For now, I am going comment out the drop database line in the 616 sql file, drop all the tables from the DBs, and re-run all the database imports.

I am not certain what further information you need apart from the hash I am using for wotlk-db and mangos-wotlk.

wotlk-db: 5536be83ece6a9b85765e556fcfe8d2ce61b19a7
mangos-wotlk: 8794ef0920e52d59cfa0c2e9466a2fd2d8f52f76

Thanks.

**Comment by Grz3s** [comment]
```quote
When the InstallFullDB.sh executes it also loads the updates one of which is 616_corepatch_12945.sql and it drops the spell_template table. Which causes mangosd to fail to start.
```
Its nothing wrong with this update...
Spells been moved into... ...(cmangossources)\\\sql\base\dbc folder
and if your InstallFullDB.sh is set correctly... you should see that Spell_template is loaded in final part of it. (this runs perfectly fine without anny issues)
BTW:
```quote
(...)causes mangosd to fail to start.
```
Pls show log.

**Comment by grentech** [comment]
@Grz3s 

Thank you for the reply.

> and if your InstallFullDB.sh is set correctly... you should see that Spell_template is loaded in final part of it. (this runs perfectly fine without anny issues)

I have discovered the source of my issues.  It appears that I did not RTFM.  I just read through InstallFullDB.sh, the dbc loading starts at line 245 (of wotlk-db 5536be83ece6a9b85765e556fcfe8d2ce61b19a7) and uses variable CORE_PATH and when I configured the .config file I did not set CORE_PATH.

Thank you for pointing that out to me ever so politely.

I apologize for erroneously submitting an issue.

At this point I would argue that showing the log is irrelevant since I definitely know what (my/) the problem is...

Thanks again for taking the time to reply.  I appreciate it.
